mct_quantizers/keras/quantize_wrapper.py

```diff
@@ -153,17 +253,38 @@ def from_config(cls, config):
             Returns: A KerasQuantizationWrapper
 
             """
+            numpy_deserialization = lambda **_config: tf.constant(**_config).numpy()
+            tensor_deserialization = lambda **_config: tf.constant(**_config)
+
+            def maybe_int(x):
+                try:
+                    return int(x)
+                except ValueError:
+                    return x
```

Comment:
If it's aimed to convert strings only, maybe use isdigit instead of relying on exceptions for the program flow?